docs/faq.md

```diff
@@ -225,7 +226,7 @@ across all instances and allows easy user control with
 
 ### What Slurm image do I use for production?
 
-By default, the [slurm_cluster](../terraform/slurm_cluster/README.md) terraform
+By default, the slurm_cluster terraform
```

Comment:
By default, Slurm clusters using Cluster Toolkit make use of the latest `slurm-gcp-6-9-hpc-rocky-linux-8` image